googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java

```diff
@@ -456,9 +456,8 @@ public void testDeleteTrainedModelWithMandatoryParameters() throws Exception {
                                 + connectorProperties.getProperty("project") + "/trainedmodels/"
                                 + connectorProperties.getProperty("modelId"), "DELETE", apiRequestHeadersMap);
         
-        Assert.assertEquals(204, esbRestResponse.getHttpStatusCode());
-        Assert.assertEquals(404, apiRestResponse.getHttpStatusCode());
-        
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), esbRestResponse.getHttpStatusCode());
+
```

Comment:
Fixed in latest https://github.com/wso2-extensions/esb-connector-googleprediction/blob/master/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java